code/modules/surgery/organs/organ.dm

```diff
@@ -217,6 +217,7 @@
 	if(tough)
 		return
 	damage = clamp(damage + amount, 0, max_damage)
+	damage = round(damage, 0.1)
```

Comment:
Would be good to make a proc for this somewhere that can be used by the rest. Maybe also make it round to `0.01` as I'm not 100% sure if `0.1` wouldn't cause weirdness with certain chems/virus symptoms/etc

Reply:
done